### ISSUE TITLE
Unable to select a tag on Chrome mobile Android v12

### ISSUE THREAD

**Issue by kerozen77127** [yairEO/tagify#1023]
We encounter a bug when the list of tags appears below the tagify fields on **Google Chrome** and **Firefox** from our smartphone.

When this happens, if we click on a tag among those proposed, it is not selected and the browser acts as if we had clicked on the element behind the list.

However, when we go further in the search for the label by entering some letters, the system works.

Here are several image captures to explain our problem.

[Work exemple 1](https://cdn.discordapp.com/attachments/245914326827925504/970729856864251904/Screenshot_2022-05-02-18-53-25-16_40deb401b9ffe8e1df2f1cc5ba480b12.jpg)
✔️ The list opens above it works

[Work exemple 2](https://cdn.discordapp.com/attachments/245914326827925504/970729857191383040/Screenshot_2022-05-02-18-52-20-45_40deb401b9ffe8e1df2f1cc5ba480b12.jpg)
✔️ The list opens at the bottom but we do a deep search it works 

[Bug exemple](https://cdn.discordapp.com/attachments/245914326827925504/970729857724072007/Screenshot_2022-05-02-18-52-06-55_40deb401b9ffe8e1df2f1cc5ba480b12.jpg)
❌The list opens at the bottom without  deep search it does not work

There is no error message in the console and it worked until recently.

Thank you very much for your help.

Best regards

**Comment by kerozen77127** [comment]
After long tests we think that the problem occurs when the tag list appears behind the keyboard, as you can see on the picture below.

[Bug exemple](https://cdn.discordapp.com/attachments/245914326827925504/970994186830630912/Screenshot_2022-05-03-12-23-53-77_40deb401b9ffe8e1df2f1cc5ba480b12.jpg
)
If I click on one tag, the list closes and nothing is selected

**Comment by yairEO** [comment]
Have you tried debugging this on PC with viewport set to mobile size?

**Comment by kerozen77127** [comment]
Yes, it works when we tried to debug on PC with mobile viewport size. The problem only appears on mobile with the keyboard open.

**Comment by yairEO** [comment]
I've just tried using my mobile Chrome and it works well on Tagify's demo page
